liqd_product/apps/partners/models.py

```diff
@@ -33,12 +34,15 @@ class Partner(models.Model):
         help_prefix=_(
             'The image will be shown as a decorative background image.'
         ),
-        upload_to='parterns/backgrounds',
+        upload_to='partners/backgrounds',
         blank=True
     )
-    about = RichTextUploadingField(
+    information = RichTextUploadingField(
```

Comment:
The information and imprint fields should be optional, as it seems that not every municipality needs these (Falkensee only wants the imprint)